ONSdigital/dp-compose
# Feature/adding config for new service

### What

Added config so that new frontend service can be added to the [cantabular import journey](https://github.com/ONSdigital/dp-compose/tree/main/cantabular-import)

### How to review

Sense check
To run the stack locally:
- Pull this branch
- Follow the instructions to setup the services
- In the `dp-filter-api` service, set the `ASSERT_DATASET_TYPE` to `false` (this gets round a problem with the recipes using the wrong property)
- Pull the [feature/add-docker-container-config](https://github.com/ONSdigital/dp-frontend-filter-flex-dataset/tree/feature/add-docker-container-config) branch of `dp-frontend-filter-flex-dataset`
- `make start`

**Note** [PR for dp-frontend-filter-flex-dataset to add the config for the journey](https://github.com/ONSdigital/dp-frontend-filter-flex-dataset/pull/9)

### Who can review

!me
